proposals/0001-dep-process.md

```diff
@@ -5,7 +5,7 @@ Short Name: `0001-dep-process`
 
 Type: Process
 
-Status: Draft (as of 2018-XX-YY)
+Status: Draft (as of 2018-01-30)
```

Comment:
I noticed all other proposals had a date, I figured this was probably just missing a date.